components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-overlay {
+  /* TODO: replace with core tokens */
+  --spectrum-overlay-animation-distance: 6px;
+  --spectrum-global-animation-duration-100: 130ms;
+
+  visibility: hidden;
+
+  opacity: 0;
+
+  transition: transform var(--spectrum-global-animation-duration-100) ease-in-out,
+              opacity var(--spectrum-global-animation-duration-100) ease-in-out,
+              visibility 0ms linear var(--spectrum-global-animation-duration-100);
+
+  pointer-events: none;
+}
+
+%spectrum-overlay--open {
+  visibility: visible;
+
+  opacity: 1;
+
+  transition-delay: 0ms;
+
+  pointer-events: auto;
+}
+
+%spectrum-overlay--bottom--open {
+  --spectrum-overlay-animation-distance: 6px;
```

Comment:
hmm, lots of repeats. maybe move this var into `components/tokens/custom-spectrum/custom-vars.css` if it's being used so much? That's our current WIP core tokens place anyway. 

I see why with the placeholder mixin that you had to re-define them.

Reply:
@GarthDB could I get your input on this? I left overlay alone while working on Popover, but it looks like we may need to do some refactoring here. Can we move `--spectrum-overlay-animation-distance` into **components/tokens/custom-spectrum/custom-vars.css**?

Also note that I'm not sure this should be **6px**. Ultimately popover needs to be **8p**x away from the source (or **16px** away when it has a tip, which can be done with a calc). But popover distance to source was deemed out of scope for popover. 

Also, I am not sure if the 6px was just leftover or if that measurement is needed for other components that are using popover. Maybe we need multiple tokens for animation distance.

Reply:
Can you define `--spectrum-overlay-animation-distance` in one place in this file and reuse it?

If there is another Jira issue that "popover distance to source" will be addressed in, perhaps further refactoring can be done there since "distance to source" is out of scope.

Reply:
I think we will need input from @GarthDB to decide if anything needs to change regarding use of overlay within popover or in general. There is an overlay component (/components/overlay) as well as this commons file (components/commons/overlay-coretokens.css) and I'm not sure if either or both of them need changes, as Garth helped me get them set up to import into popover and I don't have any direction as to how to handle overlay in general.

Reply:
Once those components are done with core tokens, we can remove the overlay component if nothing else is using it.